spec/support/custom/common_actions/documents.rb

```diff
@@ -0,0 +1,8 @@
+module Documents
+  def documentable_fill_new_valid_budget_investment
+    fill_in "Title", with: "Budget investment title"
+    fill_in_ckeditor "Description", with: "Budget investment description"
+    fill_in "Estimated price (€)", with: "10000"
+    check :budget_investment_terms_of_service
+  end
```

Comment:
What do you think about:

```suggestion
    alias_method :consul_documentable_fill_new_valid_budget_investment, :documentable_fill_new_valid_budget_investment

  def documentable_fill_new_valid_budget_investment
    consul_documentable_fill_new_valid_budget_investment

    fill_in "Estimated price (€)", with: "10000"
  end
```

Same in the `Images` and `Maps` modules.